Moodis/Moodis/Network/Face/Face.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moodis.Network.Face
+{
+    public sealed class Face
+    {
+        private Face(){}
+
+        private const string SubscriptionKey = "e285e7e510e0493bbaa7ffe5ffc2db77";
```

Comment:
Got an email for your PR, so this is a random comment :D please, never ever store any secrets in git. Try looking up on how to store secrets in win forms.

Reply:
Since this key is temporary(valid only for 7 days), we just saved it as a simple constant for now. We will later on implement a more secure solution for storing secrets.